### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by deeplook** [quantopian/qgrid#348]
I see a significant number of PRs and issues with people making the effort to provide the desired details, but not much feedback or other activity.

**Comment by JohnOmernik** [comment]
Agreed. More information would be nice to know, especially with JupyterLab 3 out, understanding the maintained status of this is important.  

Thanks!

**Comment by adrienpacifico** [comment]
If not maintained, what is the current best alternative?

**Comment by amimoto** [comment]
I don't think it's maintained anymore... Quantopian appears to have gone defunct in 2020
https://en.wikipedia.org/wiki/Quantopian

Does anyone know of an active fork?

**Comment by deeplook** [comment]
I'd say that https://github.com/bloomberg/ipydatagrid is an awesome alternative...

**Comment by amimoto** [comment]
> I'd say that https://github.com/bloomberg/ipydatagrid is an awesome alternative...

That one is awesome. Thank you for the suggestion!

**Comment by pankajp** [comment]
I see some activity on the forks by @8080labs ( https://github.com/8080labs/ipyslickgrid ) which got acquired by Databricks, so unclear future; and by @modin-project  ( https://github.com/modin-project/modin-spreadsheet ) , which has lots of complicated dependencies. Here's some issues i am aware of:
 - https://github.com/bloomberg/ipydatagrid : loads all data in ui (no virtual scroll display) so slow with very large dataframes
 - https://github.com/8080labs/ipyslickgrid : uncertain future, i had some issues with filter display css
 - https://github.com/modin-project/modin-spreadsheet : failed to run with all the dependencies needed

**Comment by JohnOmernik** [comment]
I have cloned qgrid and spent a day learning it well enough to work with an up-to-date Jupyterlab.  I prebuilt the extension and it seems to be working. I am not an expert JS coder, and more of a hacker, but I got it working. If we move over here, we could start maybe cleaning up some of the bugs in the code, but I will need other coders to help!  

https://github.com/JohnOmernik/qgrid

**Comment by vthemelis** [comment]
Thanks for the research @pankajp 

I saw `modin-spreadsheet` but it looks like they are tying it to the rest of their ecosystem (eg, you need to import `modin.pandas` instead of just `pandas`).

I'm more of a fan of `ipyslickgrid` but I see that the project has disabled the `Issues` tab.

**Comment by zhihanyue** [comment]
I've created a new repo [QgridNext](https://github.com/zhihanyue/qgridnext) to continue maintaining it. QgridNext is now compatible with recent releases of Jupyter and addressed many bugs found in Qgrid.

**Comment by chrowe** [comment]
https://github.com/mwouts/itables seems to be the most popular alternative at this point, with https://github.com/bloomberg/ipydatagrid a close second.

I like the fact that ipydatagrid is maintained by Bloomberg, so it should be around for a while.

After testing I think I am going to use itables because it comes with a simple search field out of the box, which is probably my most common use case.
